sbfl/utils.py

```diff
@@ -0,0 +1,210 @@
+import os
+import re
+import pandas as pd
+from . import base
+
+def is_line_coverage(l: str) -> bool:
+    m = re.match(r"^\s+\S+:\s+\d+:", l)
```

Comment:
I've test with a source code with 100,012 lines.

Unfortunately, this regex cannot match following lines :smile_cat: 
(Please refer a file format of `gcov` at https://man7.org/linux/man-pages/man1/gcov.1.html )
```
        1:99999:    b++;   
        1:100000:    b++;   
        1:100001:    b++;   
        1:100002:    b++;   
        1:100003:    b++;   
        1:100004:    b++;   
        1:100005:    b++;   
        1:100006:    b++;   
        1:100007:    b++;   
        1:100008:    b++;   
        1:100009:    b++;   
        1:100010:    b++;   
        1:100011:    return ++b;
        -:100012:}
```

But I'd like to suggest to merge this PR at this moment.
After preparing some test cases, we may fix this line with a confidence.

Reply:
Thank you for reporting this!
I'll fix this and add some test cases as soon as possible.